kmk/handlers/stock.py

```diff
@@ -102,7 +102,7 @@ def bkdl_released(key, keyboard, KC, *args, **kwargs):
 
 
 def sleep_pressed(key, keyboard, KC, *args, **kwargs):
-    sleep_ms(key.meta.ms)
+    ticks_ms(key.meta.ms)
```

Comment:
did this actually get fixed? My testing didn't exercise this area of the code, but I think calling ticks_ms() with an argument is incorrect.

Reply:
It was, in fact broken. This solves it.